### PR TITLE
Removes ways for players to become/create antagonists from ruins

### PR DESCRIPTION
**Pull request by Zevotech**
## About The Pull Request
Removes a romerol bottle from a space ruin, a few facehugger eggs from a jungle ruin, replaces the morph syringe inside of the gluttony ruin on lavaland to a cleaving saw and a mobility scooter and outright removes the wishgranter cube ruin.

## Why It's Good For The Game
The ability to create or become antagonists as powerful as xenomorphs, zombies and the goddamn wishgranter should be relegated solely to admins, not the players. Admins should not be REQUIRED to deal with a xenomorph outbreak or a murderboning morph, but sometimes they have to anyways. 
If a xenomorph outbreak starts, it should be started and overseen by an admin, not because some scientist wanted to try and harvest xenomorph organs for fun.
If a romerol zombie outbreak starts, same thing. An admin shouldnt have to punish a dumb player because a mapper gave them something they shouldnt use anyway.
If a morph is murderboning because they found the gluttony ruin, theyre just stupid. Everyone knows a morph is only good for playing prop hunt.
The wishgranter shouldnt exist, and should never have to exist on this codebase without adminbus.

Even if this PR does stir up debate, you can't deny that players shouldn't be allowed to create antagonists without an admin giving them the go-ahead just because someone put a xenomorph egg or bottle of romerol on a ruin.

## Changelog
:cl:
del: The wishgranter cube ruin has vanished from existence... for now
balance: removed the bottle of romerol from ntfacility.dmm
balance: replaced the three facehugger eggs in jungle_nest.dmm with pre-burst ones
balance: replaced the "gluttonys blessing" syringe in lavaland_surface_gluttony.dmm with a cleaving saw and mobility scooter
/:cl:

### CODE REVIEW

File: _maps/RandomRuins/LavaRuins/lavaland_surface_gluttony.dmm [hunk 1]
```diff
@@ -12,7 +12,11 @@
 /turf/open/lava/smooth,
 /area/ruin/powered/gluttony)
 "f" = (
-/obj/item/reagent_containers/syringe/gluttony,
+/obj/item/melee/transforming/cleaving_saw,
+/obj/vehicle/ridden/scooter{
+	desc = "After eating that much junk, you will probably need this... If you can even get it out of this room!";
```
Comment (Fikou) [inline _maps/RandomRuins/LavaRuins/lavaland_surface_gluttony.dmm:17]: Did you test whether or not you can get it out of the room

**Comment by zestybastard** [comment]
I would like to go on the record as one of Shiptest's head administrators to say that, while the intentions of this PR are appreciated, it actually accomplishes the opposite of its goal. Let me explain:

Ways for players to naturally get antagonist status or create antagonists are extremely rare at the moment, meaning that players often rely on administrators to make those things happen. However, we don't appreciate the level of control it offers comparable to the fact it's a chore to set these things up ourselves sometimes. As a result, ways for players to create antagonists naturally are actually in **good standing** with the administrative community right now.

If there was one thing I would suggest, it's finding some new ways to add antags into the game that can just show up sometimes, so players don't feel pressured to ask admins when they want this sort of thing.

**Comment by genessee-forgot-his-password-again** [comment]
Hello, shiptest admin here. I like the antags that can spawn right now and so far most situations arising from them have been fun to observe and haven't caused much uproar, if any, in tickets.

Would rather the means to create antags stay in so I have less people clamoring for 'free anteg' every round. Thank you.

**Comment by triplezeta** [comment]
considering the overall response from the administration team and the basis of this pr, i don't think anyone will oppose this

**Comment by Zevotech** [comment]
alright cool

**Pull request closed without merging**